server/content-model/events.js

```diff
@@ -4,103 +4,113 @@ import type {Person} from './people';
 import type {Picture} from '../model/picture';
 
 export type DateTimeRange = {|
-  startDateTime: Date;
-  endDateTime: Date;
+  startDateTime: Date,
+  endDateTime: Date
 |}
 
 type ContributorRole = {|
-  id: string;
-  title: string;
+  id: string,
+  title: string
 |}
 
 export type Contributor = {|
-  role: ContributorRole; // TODO: Should this just be a free text string?
+  role: ContributorRole;
```

Comment:
This could be a comma while you're at it?

Reply:
Could be and is.